tools/config_updater.py

```diff
@@ -696,6 +740,91 @@ def ManageBinaries(config=None, token=None):
   print "Please restart the service for it to take effect.\n\n"
 
 
+def SuggestedConfiguration():
```

Comment:
This is good information, but I don't think putting it in code is the right thing to do. Lets add a documentation page and reference it in the "initialization complete message"